src/ConfigFactory.Avalonia/ConfigPage.axaml

```diff
@@ -66,6 +66,7 @@
                         Margin="5,0"
                         HorizontalAlignment="Stretch"
                         HorizontalContentAlignment="Center"
+                        Command="{Binding SecondaryRelayCommand}"
```

Comment:
Glad you spotted this one, had it fixed locally and forgot to commit it :D

Reply:
I noticed that the Cancel button does nothing and that the `Reset()` method is suspiciously nowhere used :)